qabelbox/src/test/java/de/qabel/qabelbox/box/interactor/BoxSharerTest.kt

```diff
@@ -97,6 +99,6 @@ class BoxSharerTest {
 
     private fun share() {
         useCase.upload(path, samplePayload.toUploadSource(sample)).waitFor()
-        sharer.sendFileShare(contact, path).waitFor()
+        sharer.sendFileShare(contact, path)
```

Comment:
How do you know that the share was already sent? At this point, it should only have been scheduled but without waiting for this observable to complete, your test are probably flaky (because the share might or might not have been sent and the assertions might or might not fail).